.circleci/config.yml

```diff
@@ -182,10 +182,57 @@ jobs:
       - store_artifacts:
           path: ~/test-results
 
+  lint:
+    docker:
+      - image: gcr.io/cloud-builders/docker
+    steps:
+      - run: apt-get install -y make
+      - checkout
+      - setup_remote_docker
+
+      - restore_cache:
+          keys:
+            - v1-sd-lint-layers-{{ .Branch }}-
+            - v1-sd-lint-layers-
+          paths:
+            - /caches/layers.tar
+
+      - run:
+          name: Load image layer cache
+          command: |
+            set +o pipefail
+            docker load -i /caches/layers.tar |true
+
+      - run:
+          name: Build Docker images
+          command: |
+            set +o pipefail
+            docker images
+            fromtag=$(docker images |grep securedrop-lint |head -n1 |awk '{print $2}')
+            EXTRA_BUILD_ARGS="--cache-from securedrop-lint:${fromtag:-latest}" make ci-lint-image
+
+      - run:
+          name: Save Docker image layer cache
+          command: |
+            mkdir -p /caches
+            docker images
+            docker save -o /caches/layers.tar securedrop-lint:${CIRCLE_SHA1}
+
+      - save_cache:
+          key: v1-sd-lint-layers-{{ .Branch }}-{{ epoch }}
+          paths:
+            - /caches/layers.tar
+
+      - run:
+          name: Run Linting
+          command: |
+            make ci-lint
+
 workflows:
   version: 2
   securedrop_ci:
     jobs:
+      - lint
       - tests
       - docs-lint
```

Comment:
since **make lint** includes **docs-lint**, this can be removed to not run it twice

Reply:
this run shows docs-lint taking a third of the time as lint and as a dep of the longer runs this could slow down total build times. https://circleci.com/workflow-run/60bea5a3-d9f3-4036-a0b5-bcf5a4290188